zipkin-lens/src/translations/zh-cn.json

```diff
@@ -17,5 +17,16 @@
   "src.components.TracePage.duration": "持续时间",
   "src.components.TracePage.services": "服务",
   "src.components.TracePage.totalSpans": "Span总数",
-  "src.components.TracePage.viewLogs": ""
+  "src.components.TracePage.viewLogs": "",
+  "src.components.DependenciesPage.dependencies": "Dependencies",
```

Comment:
Did you copy in the English strings here? It's better to leave empty string (should be what the extract messages tool did) which falls back to the default which is English. Otherwise it's harder to know what strings are pending translation.